packages/mongo-join-builder/package.json

```diff
@@ -10,10 +10,10 @@
   "dependencies": {
     "@keystonejs/utils": "^5.1.0",
     "cuid": "^2.1.6",
-    "lodash.omitby": "^4.6.0"
+    "lodash.omitby": "^4.6.0",
+    "mongodb-memory-server": "6.1.1"
   },
   "devDependencies": {
-    "mongodb": "^3.3.3",
-    "mongodb-memory-server": "^6.0.1"
+    "mongodb": "^3.3.3"
```

Comment:
Is there a particular reason for moving this out of `devDependencies`?

Reply:
My bad, will fix it ASAP